.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "brightspace/polymer-config",
+  "env": {
+    "es6": true
```

Comment:
Thinking maybe we should just go ahead and add this to our shared polymer eslint config.